src/js-api/handlers/login.ts

```diff
@@ -0,0 +1,18 @@
+import { EndpointGetMyCustomerOrgResBody } from "@baseplate-sdk/web-app";
+import { createBaseplateFetch } from "../../cli-utils";
+import { log, success } from "../../cli-logger";
+import { storageProvider } from "../../cli-storage";
+
+const storage = await storageProvider();
```

Comment:
Top-level await is something I generally have avoided in my code, since it delays things.

I'd prefer moving this inside of the `login` function, to avoid the performance consequences of executing this file when the user isn't running the `baseplate login` command